src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="first"
+    action="/"
+    method="get"
+    data-qa="big"
+    >
```

Comment:
do mind for proper indentation across your code - that is basics.
```suggestion
    <form
      class="first"
      action="/"
      method="get"
      data-qa="big"
    >
```